src/mock_adapter/controllers/bag.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import appier
+
+import models
+
+class BagController(appier.Controller):
+
+    @appier.route("/bags/me", "GET")
+    def me_bag(self):
+        session_id = 1
+        bag = models.Bag.get(session_id = session_id, map = True)
+        return bag
+
+    @appier.route("/bags/<int:id>/add/<int:product_id>", "POST")
+    def add_to_bag(self, id, product_id):
+        quantity = self.get_field("quantity", cast = int)
+        session_id = 1
+        product = models.Product.get(id = product_id)
+        bag = models.Bag.get(session_id = id)

```

Comment:
create method in bag called get_from_session(), this method returns the session bag